### ISSUE TITLE
Feature Request: Add option to display token artwork instead of character artwork in chat messages

### ISSUE THREAD

**Issue by nschoenwald** [foundryvtt/dnd5e#3077]
Currently, the character artwork is always used in chat messages. That makes certain scenarios very challenging.
To be more specific, I have an adventure where scarecrows are disguised as humans. I have one scarecrow actor and different token images for their human appearance. When I roll something with the disguised scarecrows, it always shows the scarecrow in the chat message.
This extends to other scenarios. Even having one "commoner" actor with different token images for NPCs, the chat messages will always show the one character artwork set on the commoner actor.
Having the option to show token artwork instead of character artwork would solve all of this.

**Comment by Rabulias** [comment]
Is this something new with DND5E 3.x? I am on 2.4.1 and I don't see character pictures in chat messages. I do see item, spell, weapon, etc., icons when an item, spell, weapon, etc., is used. Maybe I am missing something?

**Comment by unsoluble** [comment]
Yes, this is a v3 feature. (You can safely assume that all open tickets now refer to v3.)

**Comment by RavenknightDM** [comment]
I supprt this request and would really want to see it implemented as soon as possible. Forcing us to use Actor portraits is a bummer.

**Comment by nschoenwald** [comment]
@Fyorl Any chance this could get added to a future milestone? This is a regular annoyance for me and leads to having to create multiple copied actors where it would not have been neccessary in earlier system versions.

**Comment by nschoenwald** [comment]
@Fyorl @arbron 

Any chance this could get added to a future milestone? This is a regular annoyance for me and leads to having to create multiple copied actors where it would not have been necessary in earlier system versions.

The chat portrait is basically the only place where you are forced to use the image or name from the "prototype" actor instead of from the token itself.

**Comment by Fyorl** [comment]
We made improvements to how `showTokenPortrait` functions in 5.1, so this should be a simple case of respecting that flag when rendering the chat avatar.
